viz_scripts/docker/cert.sh

```diff
@@ -0,0 +1,6 @@
+if [ -z ${PROD_STAGE} ] ; then
+    echo “Not in staging / production environment, continuing build...” 
```

Comment:
This script allows us to eliminate a major difference between internal and external repos by allowing an environment variable to control whether or not we download the certificate.

Reply:
I see. However, I don't think this check is necessary. The certificate bundle is public so it is not sensitive information. Having the certs in stage/production doesn't harm anything, they are just an extraneous file which will be sitting around doing nothing.

And removing the check and always copying the file will make this much simpler - we don't need to have a separate environment variable that is set sometimes and not at other times.

Reply:
I can delete cert.sh and add a line to the dockerfile of viz-scripts to copy the file, just like how it is in the internal repo. I'll also remove the code where the script is run from.